### ISSUE TITLE
getPolicyId and getResourceId not working correctly if there are >100 entities in realm

### ISSUE THREAD

**Issue by test4423** [adorsys/keycloak-config-cli#1127]
### Current Behavior

When getPolicyId or getResourceId are called, they only search the first 100 policies/resources because they call Keycloak API without "max" parameter:

ClientRepository.java:
```
private String getPolicyId(ClientResource clientResource, String policyName) {
  return clientResource.authorization().policies().policies() // no parameters
```
```
private String getResourceId(ClientResource clientResource, String resourceName) {
  return clientResource.authorization().resources() // no parameters
```

The following code seems to work fine:

```
private String getPolicyId(ClientResource clientResource, String policyName) {
  return clientResource.authorization().policies().policies(null, null, null, null, null, null, null, null, -1, -1)
```
```
private String getResourceId(ClientResource clientResource, String resourceName) {
   return clientResource.authorization().resources().find(null, null, null, null, null, -1, -1)
```
Perhaps even findByName() could be used there, but I didn't test it.

### Expected Behavior

These methods should search all policies/resources

### Environment

- Keycloak Version: 25.0.1
- keycloak-config-cli Version: 6.1.7
- Java Version: 21

**Comment by bohmber** [comment]
@test4423 @francis-pouatcha Could be fixed by #1096

**Comment by francis-pouatcha** [comment]
@test4423 #1096 was merged. Can you please retest?
